src/utils/RegHubUtil.js

```diff
@@ -25,7 +26,7 @@ module.exports = {
     return obj;
   },
 
-  search: function (query, page) {
+  search: function (query, page=1) {
```

Comment:
Haha, tbh I threw it in there out of coding habit and was pleased that it worked - Totally forgot regular JS doesn't like it.